std/format.d

```diff
@@ -4391,22 +4391,27 @@ if ((is(T == struct) || is(T == union)) && (hasToString!(T, Char) || !is(Builtin
         put(w, left);
         foreach (i, e; val.tupleof)
         {
-            static if (0 < i && val.tupleof[i-1].offsetof == val.tupleof[i].offsetof)
-            {
-                static if (i == val.tupleof.length - 1 || val.tupleof[i].offsetof != val.tupleof[i+1].offsetof)
-                    put(w, separator~val.tupleof[i].stringof[4..$]~"}");
-                else
-                    put(w, separator~val.tupleof[i].stringof[4..$]);
-            }
+            static if (__traits(identifier, val.tupleof[i]) == "this")
+                continue;
             else
             {
-                static if (i+1 < val.tupleof.length && val.tupleof[i].offsetof == val.tupleof[i+1].offsetof)
-                    put(w, (i > 0 ? separator : "")~"#{overlap "~val.tupleof[i].stringof[4..$]);
+                static if (0 < i && val.tupleof[i-1].offsetof == val.tupleof[i].offsetof)
```

Comment:
Please make this an `else static if` rather than introduce an extra pair of braces.